spec/expeditor/service_spec.rb

```diff
@@ -89,16 +89,18 @@
   end
 
   describe '#status' do
-    let(:service) { Expeditor::Service.new(sleep: 10) }
     it 'returns current status' do
+      # Set large value of period in case test takes long time.
+      service = Expeditor::Service.new(period: 100)
```

Comment:
>to clarify the test target

Aha, I should take the advantage too, but let me leave these "without `let` code" as they are here.